unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs

```diff
@@ -180,7 +180,7 @@ public void Update_Trigger(InputAction_Trigger[] triggerTimeActions)
                     InputProcessor.FromKey(action, KeyCode.U, modifiers: InputProcessor.Modifier.None);
                     break;
                 case DCLAction_Trigger.CloseWindow:
-                    if (allUIHidden || DataStore.isSignUpFlow.Get()) break;
+                    if (allUIHidden || DataStore.i.isSignUpFlow.Get()) break;
```

Comment:
if (allUIHidden || DataStore.i.isSignUpFlow.Get())
                      break;

Reply:
I dont think we have a convention against one-liners but will change it anyway! Thx!